Sources/Testing/Support/Graph.swift

```diff
@@ -676,6 +676,49 @@ extension Graph {
     try await compactMapValues(transform)!
   }
 
+  /// Create a new graph containing the nodes of this graph with the values
+  /// transformed by the given closure, with the option to recursively apply
+  /// the result of that transformation to all descendants of each node.
+  ///
+  /// - Parameters:
+  ///   - transform: A closure that is invoked once per element in the graph.
+  ///     The element representing that node is passed to this closure and its
+  ///     result is used as the corresponding value in the new graph.
+  ///
+  /// - Returns: A graph containing the transformed nodes of this graph at the
+  ///   same key paths.
+  ///
+  /// - Throws: Whatever is thrown by `transform`.
+  ///
+  /// This function iterates depth-first.
+  func mapValues<U>(_ transform: (Element) async throws -> U) async rethrows -> Graph<K, U> {
```

Comment:
It is intentional that we don't include the key path or key in the existing `mapValues()` definition (same as for `Dictionary`.) Can you elaborate why you need it here?

Reply:
I'm modifying all the mapValues APIs to accept key path in #372